lib/msf/base/sessions/command_shell_options.rb

```diff
@@ -15,6 +15,12 @@ module CommandShellOptions
   def initialize(info = {})
     super(info)
 
+    register_options(
+      [
+        OptBool.new('CREATE_SESSION', [false, 'Create a new session on successful login.', true])
```

Comment:
Honestly, this should probably be an advanced option like `CreateSession`.

Reply:
Done